clientd3d/d3drender_skybox.h

```diff
@@ -14,9 +14,30 @@
 #ifndef _D3DRENDERSKYBOX_H
 #define _D3DRENDERSKYBOX_H
 
-void D3DRenderSkyBoxBegin();
-void D3DRenderSkyBox(room_type* room, Draw3DParams* params, room_contents_node* pRNode, 
-	int& angleHeading, int& anglePitch);
+struct skybox_render_object {
```

Comment:
Maybe change the name, since "object" doesn't mean anything.  We have Draw3DParams, so perhaps SkyboxRenderParams?